systemd/manageiq.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=ManageIQ
+After=syslog.target
```

Comment:
This is an interesting `After=` directive, syslog is super early afaik, we might want to look at using something else (like a manageiq-vmdb.target which tests the pg connection or something)